### PR TITLE
Plot: Fermi—Dirac temperature dependence

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+occupancy = distribution_law.occupancy_of_state
+energy = distribution_law.energy_of_state
+chemical_potential = distribution_law.total_chemical_potential
+temperature = distribution_law.temperature
+
+reduced_temperature = symbols("reduced_temperature")
+reduced_temperature_eqn = Eq(
+    reduced_temperature,
+    units.boltzmann_constant * temperature / (energy - chemical_potential),
```
Comment (blackyblack) [inline plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py:19]: Law for this?

File: plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    title="Occupancy as a function of reduced temperature for Fermi—Dirac distribution",
+    xlabel=r"$\frac{k_\text{B} T}{\varepsilon - \mu}$",
+    ylabel=r"$\bar n$",
+    backend=MatplotlibBackend,
+    show=False,
+)
+
+asymptote_expr = distribution_expr.limit(reduced_temperature, S.Infinity)
+
+asymptote_plot = plot(
```
Comment (blackyblack) [inline plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py:42]: I suggest to add legend for the plot. It is not clear, what is this line.

File: plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py (hunk 1 above)
Comment (alesanter) [reply plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py:19]: I don't think that's needed. This is only used once for the plot

File: plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py (hunk 2 above)
Comment (alesanter) [reply plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py:42]: Agree

File: plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py (hunk 1 above)
Comment (blackyblack) [reply plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py:20]: Well it looks rather interesting for a separate law. It is a parameter of plot but I do not see where it comes from.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**